### ISSUE TITLE
Review tests for old versions of Sass

### ISSUE THREAD

**Issue by romaricpascal** [alphagov/govuk-frontend#5541]
## What

Review what confidence tests checking deprecated versions of Sass, especially the oldest version, bring to us and if anything needs updating in that space.

## Why

We've had recent issues with the tests against the oldest LibSass package:
- https://github.com/alphagov/govuk-frontend/pull/5491
- https://github.com/alphagov/govuk-frontend/pull/5521
- https://github.com/alphagov/govuk-frontend/pull/5522 

There's also been issues slipping past us when building with LibSass: https://github.com/alphagov/govuk-frontend/issues/5415.

## Who needs to work on this

Developers

## Who needs to review this

Developers

## Done when

- [ ] We've reviewd the tests, why they're here and what they bring to us
- [ ] We've created actions from the review as issues on GitHub

**Comment by 36degrees** [comment]
I don't believe this is relevant any more as we now only have tests for Dart Sass (oldest and latest versions we support)
